dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -120,6 +121,7 @@ def __init__(
         self.environment = environment or {}
         self.tags = tags
         self._find_address_timeout = find_address_timeout
+        self._use_private_ip = use_private_ip
```

Comment:
Other parameters seem to have a one-line description in the docstring here as well

Reply:
Thanks, fixed! (And noticed `name` didn't have a description so added that, too.)